web/packages/teleport/src/components/ToolTipNoPermBadge/ToolTipNoPermBadge.tsx

```diff
@@ -55,10 +61,10 @@ export const ToolTipNoPermBadge: React.FC<Props> = ({
           background-color: ${p => p.theme.colors.error.main};
         `}
       >
-        Lacking Permissions
+        {tooltip}
       </Box>
       <Popover
-        modalCss={() => `pointer-events: none;`}
+        modalCss={() => `pointer-events: ${popoverPointerEvent}`}
```

Comment:
nit: simple enough for inline
```suggestion
        modalCss={() => `pointer-events: ${sticky ? 'auto' : 'none'}`}
```